### ISSUE TITLE
Is it possible to send mails using resend if i dont have a personal domain but just my email id?

### ISSUE THREAD

**Issue by karthik11135** [resend/resend-node#454]
About a month ago, when I used resend sdk, I noticed that I wasn't able to send mails unless I have a personal domain. Is that still the case?

**Comment by emiliosheinz** [comment]
Based on the documentation, you cannot send emails using Resend without having a verified domain. You can find more about it here: https://resend.com/docs/dashboard/domains/introduction#verifying-a-domain.

**Comment by karthik11135** [comment]
is not possible for development purposes also?

**Comment by emiliosheinz** [comment]
Yes, as [here](https://resend.com/docs/send-with-nodejs), at the very beginning of the documentation, it says that having an **API key** and **verifying your domain** is a prerequisite to move forward.

**Comment by vcapretz** [comment]
hey @karthik11135, one small caveat is that for development you can send emails from `onboarding@resend.dev` but to your own email only, if you try sending from `resend.dev` to other emails you'll get an error. but that should be enough for you to test and see the flow working 😄

**Comment by YashB021** [comment]
Why my resend api is giving an error of example.com not verfied when I am using onboarding@resend.dev for email
